configure-system-libraries.sh

```diff
@@ -16,39 +16,22 @@ patch_config()
 	REPLACE=$4
 	SEARCH=$5
 
-	if command -v mono >/dev/null 2>&1 && [ "$(grep -c .NETCoreApp,Version= bin/OpenRA.dll)" = "0" ]; then
-		# Exit early if the file has already been patched
-		grep -q "target=\"${REPLACE}\"" "${CONFIG}" || return 0
-
-		printf "Searching for %s... " "${LABEL}"
-		for DIR in ${SEARCHDIRS} ; do
-			for LIB in ${SEARCH}; do
-				if [ -f "${DIR}/${LIB}" ]; then
-					echo "${LIB}"
-					sed "s|target=\"${REPLACE}\"|target=\"${DIR}/${LIB}\"|" "${CONFIG}" > "${CONFIG}.temp"
-					mv "${CONFIG}.temp" "${CONFIG}"
-					return 0
-				fi
-			done
-		done
-	else
-		# .NET 6 does not support .config files, so we must use symlinks instead
-		# Exit early if the symlink already exists
-		if [ -L "bin/${REPLACE}" ]; then
-			return 0
-		fi
+	# .NET 6 does not support .config files, so we must use symlinks instead
```

Comment:
`.NET 8` - assuming the comment still holds in 8.

Reply:
With mono gone we don't have .config files at all, so this whole file needs to be updated (comments at top, function names, etc) to reflect .NET.